doc/child-object-creation.md

```diff
@@ -0,0 +1,33 @@
+# Admin object's management - create a child object
+
+## Usage
+> Objects of the Admin can be configured :
+>   * By setting particular **attributes** to the object
+>   * By associating **children** to the object
+>
+> Children can be added, deleted, reordered in the list. Also, it's possible to configure it in a special way for the parent object, by setting children attributes for all the children. 
+>
+> This documentation explains how to associate a child to a parent object. 
+
+### Create a new object for your child in the admin
+> Information is available for object's creation: [Modular step management](https://github.com/01-edu/public/blob/master/doc/object-creation.md)
+
+### Add this new object as a child of your parent's object
+
+#### 1. Edit the parent object
+<img width="640" alt="Capture d’écran 2019-04-22 à 19 24 23" src="https://user-images.githubusercontent.com/35296671/56517407-cb98bc00-6534-11e9-98d6-a2b1c0193a38.png">
+<img width="640" alt="Capture d’écran 2019-04-22 à 19 24 10" src="https://user-images.githubusercontent.com/35296671/56517421-d0f60680-6534-11e9-86ef-97fb9e59786e.png">
+
+#### 2. Go to *Children* > *Add a child*
+![add child to parent object](https://user-images.githubusercontent.com/35296671/56506977-de50c800-6517-11e9-9c71-d19a1ec4e5cd.png)
+
+#### 3. Set up the new child:
+* Enter its name in the input "Add a child name"
+* Select your step object in the select input
+* Click on "ADD"
+
+> Your step is then related to its parent. You can see it in the *Children* section of the parent's object. There, you can now: 
+> * Delete the child from its parent (the actual object of your child will not be deleted).
+> * Reorder it in the children's list, by dragging it to the place you want.
+> * Go to update its settings by clicking on the eye of its reference (it will redirect you to the actual object of your step).
```

Comment:
may be worth it to precise which settings. Also you mention the steps, but I think this page of documentation is not about the onboarding steps but only about the creation of a child, could be confusing. I would for something like so. 
Update its *original* settings by clicking on the eye icon of its reference (redirection to object edit page of the child).

Reply:
updated